doc/tutorial/text_analytics/working_with_text_data.rst

```diff
@@ -311,7 +316,7 @@ Building a pipeline
 -------------------
 
 In order to make the vectorizer => transformer => classifier easier
-to work with, ``scikit-learn`` provides a ``Pipeline`` class that behaves
+to work with, ``scikit-learn`` provides a :class:`Pipeline` class that behaves
```

Comment:
this won't work, You should either change currentmodule or try sklearn.pipeline.Pipeline